Assets/Scripts/DeliveryExperiment.cs

```diff
@@ -2503,13 +2549,18 @@ private void WorldScreen()
         regularCamera.enabled = true;
         blackScreenCamera.enabled = false;
         // TODO: JPB: Hospital decide
-        //if (HOSPITAL_COURIER && STAR_SYSTEM_ACTIVE)
+        //if (EFR_COURIER && STAR_SYSTEM_ACTIVE)
         //    starSystem.gameObject.SetActive(true);
         memoryWordCanvas.SetActive(false);
         playerMovement.Zero();
```

Comment:
Should this have an `EnablePlayerTransfromReporting(true);`?

Reply:
No, I want to log positions when they are actually moving (so in between pointing and arriving at the store). When I turn it on whenever we call WorldScreen(), it would log positions during pointing, which is no bueno.

Reply:
got it